tests/queries/1_stateful/00170_s3_cache.sql

```diff
@@ -2,6 +2,7 @@
 
 -- { echo }
 
+SET enable_filesystem_cache_on_write_operations=0;
```

Comment:
Will change back to 1, when https://github.com/ClickHouse/ClickHouse/pull/36171 is merged.
Disabling this setting for now because looks like (not really sure) it might cause hung check in stress test.
This should be fixed in https://github.com/ClickHouse/ClickHouse/pull/36171 as it makes write-through cache easier and safer to use.

Reply:
> Will change back to 1, when #36171 is merged. Disabling this setting for now because looks like (not really sure) it might cause hung check in stress test. This should be fixed in #36171 as it makes write-through cache easier and safer to use.

00170_s3_cache.reference should also be modify? Otherwise it may fail the test？

Reply:
The `.reference` of this test will not be affected after disabling this setting.
But thanks for suggesting.

Reply:
> The `.reference` of this test will not be affected after disabling this setting. But thanks for suggesting.

Are you sure? The result may not be affected, but it will be in the future This instruction is output in reference.

Reply:
ah indeed!